addons/point_of_sale/models/pos_config.py

```diff
@@ -239,7 +239,7 @@ def _check_company_invoice_journal(self):
     @api.constrains('company_id', 'journal_ids')
     def _check_company_payment(self):
         if self.env['account.journal'].search_count([('id', 'in', self.journal_ids.ids), ('company_id', '!=', self.company_id.id)]):
-            raise ValidationError(_("The company of a payment method is different than the one of point of sale"))
+            raise ValidationError(_("The method payments and the point of sale must belong to the same company"))
```

Comment:
Once again, a . is missing : "The method payments and the point of sale must belong to the same company."